static/js/entry/public.js

```diff
@@ -1,5 +1,5 @@
 /* global SETTINGS: false $:false jQuery: false CURRENT_PAGE_URL: false */
-__webpack_public_path__ = `http://${SETTINGS.host}:8078/`;  // eslint-disable-line no-undef, camelcase
+__webpack_public_path__ = `${SETTINGS.public_path}/`;  // eslint-disable-line no-undef, camelcase
```

Comment:
Can we add the trailing slash in the Python side and remove it here and in the other places we define `__webpack_public_path__`?